### ISSUE TITLE
Group messaging query using DIDcomm-messaging

### ISSUE THREAD

**Issue by Shahriar707** [decentralized-identity/didcomm-messaging#454]
In an existing bifold wallet, they leverage the DIDComm protocol for messaging. Now our query is: is it possible for us to create a system that connects 3 or more devices and build a group messaging system like the popular social media WhatsApp group chat feature using the existing DIDcomm messaging protocol? Is it possible to use DIDcomm to initiate group messaging?

**Comment by swcurran** [comment]
Work was done on this by @karimStekelenburg, @genaris, and others at Rebooting Web of Trust in The Hague, NL in fall 2022. I recall there was code produced and a really cool demo, but I’m not sure where that code is.

Paper from the event: https://github.com/WebOfTrustInfo/rwot11-the-hague/blob/master/draft-documents/scalable-anywise-didcomm.md

**Comment by TelegramSam** [comment]
There has also been some work done by @frostyfrog along the lines of a server managed group. (There are a few types of group messaging, each with different characteristics...)

**Comment by frostyfrog** [comment]
Yep, and that work is in the process of being open sourced as well x3

The UI is already open source. I'm just waiting to get some time to make the back end configurable before that is opened as well. Shipping docker containers with the same secret keys isn't the best idea in the world.

**Comment by sairanjit** [comment]
Do we have a sample implementations or POCs present ?
